src/plugins/WebOfTrust/util/SynchronousDelayedBackgroundJob.java

```diff
@@ -0,0 +1,231 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * Implementation of delayed background jobs where {@link #triggerExecution} blocks until the
+ * background job has finished, as if the background job was performed synchronously. This
+ * class is intended to be used in unit tests, and is likely not very useful for general purposes.
+ *
+ * @author bertm
+ */
+public class SynchronousDelayedBackgroundJob implements DelayedBackgroundJob {
```

Comment:
To speed up the release, I've fixed this myself in commit b5f7d2c7c358234e46d4e1220a63d81b08f971ae

-> Done.